### PR TITLE
Total page overhaul

### PR DESCRIPTION
**Pull request by AgataBialoskorska**
* CSS overhaul
* remove TailwindCSS
* page structure rework
* animation rework (as a component)
* RWD adjustments (no background duplication etc.)
* gallery minification

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,13 +14,13 @@
     "e2e:headless": "start-server-and-test dev http://localhost:3000 \"cypress run --e2e\""
   },
   "dependencies": {
-    "@types/node": "20.8.6",
+    "@types/node": "20.8.0",
```
Comment (JakubBialoskorski) [inline package.json:17]: please change to 20.8.6

File: package.json [hunk 2]
```diff
@@ -14,13 +14,13 @@
...
     "@types/react": "18.2.24",
-    "@types/react-dom": "18.2.13",
+    "@types/react-dom": "18.2.7",
```
Comment (JakubBialoskorski) [inline package.json:19]: please change to 18.2.13

File: package.json [hunk 3]
```diff
@@ -14,13 +14,13 @@
...
     "eslint": "8.49.0",
-    "eslint-config-next": "13.5.4",
-    "next-sitemap": "4.2.3",
+    "eslint-config-next": "13.5.3",
```
Comment (JakubBialoskorski) [inline package.json:21]: please change to 13.5.4

**Review by JakubBialoskorski — CHANGES_REQUESTED** [review]
Titles from Head are missing, thus tests are failing.
Please align package.json to avoid regression.

File: package.json (hunk 1 above)
Comment (AgataBialoskorska) [reply package.json]: fixed

File: package.json (hunk 2 above)
Comment (AgataBialoskorska) [reply package.json]: fixed

File: package.json (hunk 3 above)
Comment (AgataBialoskorska) [reply package.json]: fixed

File: pages/projects/ios/cymbalki/index.tsx [hunk 4]
```diff
@@ -20,7 +20,7 @@ export default function CymbalkiSupportPage() {
                     <div className="w-full p-1 md:p-2">
                         <div className="flex flex-wrap w-1/3">
                             <div className="w-full p-1 md:p-12">
-                                <Link href="https://apps.apple.com/us/app/dot-xylophone/id1544977962"><img alt="gallery" className="block object-cover object-center w-full h-full rounded-lg"
+                                <Link href="https://apps.apple.com/pl/app/dot-xylophone/id1544977962"><img alt="gallery" className="block object-cover object-center w-full h-full rounded-lg"
```
Comment (JakubBialoskorski) [inline pages/projects/ios/cymbalki/index.tsx:23]: please change "pl" to "en" in the URL

File: pages/projects/ios/minspeed/index.tsx [hunk 5]
```diff
@@ -20,7 +20,8 @@ export default function MinSpeedSupportPage() {
                     <div className="w-full p-1 md:p-2">
                         <div className="flex flex-wrap w-1/3">
                             <div className="w-full p-1 md:p-12">
-                                <Link href="https://apps.apple.com/us/app/minspeed/id6470914321"><img alt="gallery" className="block object-cover object-center w-full h-full rounded-lg"
+                                {/* replace link below after deployment */}
```
Comment (JakubBialoskorski) [inline pages/projects/ios/minspeed/index.tsx:23]: Insdead of comment, it should be `https://apps.apple.com/us/app/minspeed/id6470914321`

File: pages/projects/ios/cymbalki/index.tsx (hunk 4 above)
Comment (AgataBialoskorska) [reply pages/projects/ios/cymbalki/index.tsx]: fixed

File: pages/projects/ios/minspeed/index.tsx (hunk 5 above)
Comment (AgataBialoskorska) [reply pages/projects/ios/minspeed/index.tsx]: fixed

**Comment by AgataBialoskorska** [comment]
Please re-run E2E tests

**Comment by JakubBialoskorski** [comment]
@AgataBialoskorska please run `npm install` to align `package.json` with `package.json.lock` file.

**Review by JakubBialoskorski — CHANGES_REQUESTED** [review]
Certificates images should be loaded from CDN (both miniatures and originals).
I suggest adding `?fm=png&auto=format` to each of the miniature - this will convert them "on the fly" into .avif files, compatible with client's browser (documentation: https://docs.imgix.com/tutorials/improved-compression-auto-content-negotiation). In case of fallback, .png's will be loaded but from the CDN - not locally.

**Review by JakubBialoskorski — APPROVED** [review]

**Pull request merged**